app/components/tab.vue

```diff
@@ -1,42 +1,41 @@
 <template>
   <div>
     <h1 class="page-title">FotoFluent</h1>
-    <SearchBox></SearchBox>
-    <Translation :translation="translation" v-if="translation"></Translation>
+    <SearchBox />
+    <Translation :translation="translation" v-if="translation" />
+    <TopSites />
   </div>
 </template>
 
 <script>
-  import axios from 'axios'
-  import _ from 'lodash'
+  import store from '../store/index.js'
```

Comment:
I think if we've already added the store to the root instance (in tab.js) then we don't need to re-import the store in the child components. It automatically makes it available to all child components (via this.$store) so i think we can remove the extra imports